website/www/site/i18n/home/pillars/en.yaml

```diff
@@ -11,6 +11,7 @@
 # limitations under the License.
 
 - id: home-pillars-title
-  translation: "All about Apache Beam"
+  translation: "Apache Beam Features"
```

Comment:
"Apache Beam features"

Reply:
As discussed on today's call, we keep title case everywhere.